ckan_cloud_operator/providers/ckan/instance/manager.py

```diff
@@ -64,6 +75,23 @@ def update(instance_id_or_name, override_spec=None, persist_overrides=False, wai
     else:
         pre_update_hook_data = deployment_manager.pre_update_hook(instance_id, instance_type, instance, override_spec,
                                                                   skip_route)
+
+        if config_manager.get('use-cloud-native-storage', secret_name=CONFIG_NAME):
+            cluster_provider_id = cluster_manager.get_provider_id()
+            if cluster_provider_id == 'gcloud':
+                provider_id = 'gcloud'
+            elif cluster_provider_id == 'aws':
+                provider_id = 's3'
+
+            bucket_credentials = instance['spec'].get('bucket', {}).get(provider_id)
+            if bucket_credentials:
+                literal = []
+                for key, value in bucket_credentials.items():
+                    literal.append(f'--from-literal={key}={value}')
+                literal = ' '.join(literal)
+
+                kubectl.call(f'create secret generic bucket-credentials {literal}', namespace=instance_id)
```

Comment:
please use config manager instead of calling kubectl create secret directly.

Reply:
done